run.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+if [ "$NODE_ENV" = "ci" ]
+then echo "starting service"
+  SITEROOT=/firearms GA_TAG_ID=UA-72527484-1
```

Comment:
Looks like BS to me. Why would baseUrl ever need to be `firearms/new-dealer` or `brp/not-arrived` or such like? It's as if the app is serving from a level above root, it's wrong.